hazelcast/src/main/java/com/hazelcast/security/IPermissionPolicy.java

```diff
@@ -46,6 +48,8 @@
      */
     PermissionCollection getPermissions(Subject subject, Class<? extends Permission> type);
 
+    boolean refreshClientPermissions(Set<PermissionConfig> permissionConfigs);
```

Comment:
Same reason as  `SecurityContext` but `IPermissionPolicy` doesn't refer `client` anywhere else so I'll change this. (But not sure about changing `SecurityContext`)